dumpling/export/dump.go

```diff
@@ -295,8 +310,24 @@ func (d *Dumper) Dump() (dumpErr error) {
 		if err = d.dumpDatabases(writerCtx, baseConn, taskIn); err != nil && !errors.ErrorEqual(err, context.Canceled) {
 			return err
 		}
-	} else {
+	} else if !d.conf.SQLConcurrent {
 		d.dumpSQL(writerCtx, baseConn, taskIn)
+	} else {
+		db, tableInfo, err := getTable(d.conf.Tables)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		meta, err := dumpTableMeta(tctx, conf, baseConn, db, tableInfo)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		if conf.Rows == UnspecifiedSize {
+			conf.Rows = 200000
```

Comment:
What about making it a global constant

Reply:
addressed in https://github.com/pingcap/tidb/pull/40553/commits/f0a4c4f69fddd5602abb0dd44bccbcedb77c8976